src/compute/src/render/reduce.rs

```diff
@@ -616,11 +628,22 @@ where
     {
         let error_logger = self.error_logger();
 
+        let output_name = format!(
+            "ReduceInaccumulable Distinct{}",
+            if std::any::TypeId::of::<R>() == std::any::TypeId::of::<()>() {
+                " [val: empty]"
```

Comment:
Uncovered, this looks interesting

Reply:
Same as above; please reopen this conversation if not true.